app/register/search.js

```diff
@@ -0,0 +1,117 @@
+const sequelize = require('../config/db')
+const { Op } = require('sequelize')
+const { sortAndGroupResults } = require('./search/sorting-and-grouping')
+const { cleanupSearchTerms } = require('./search/search-terms')
+const { mapResults } = require('./search/search-results')
+const { fuzzySearch } = require('../repos/search-match-codes')
+const { trigramSearch } = require('../repos/search-tgrams')
+const { rankResult } = require('../repos/search-rank')
+const { buildTsVectorQuery } = require('./search/search-builder')
+
+const trigramQueryThreshold = 0.4
+const trigramRankThreshold = 1.001
+const fuzzyRankThreshold = 1.001
+const fullTextRankThreshold = 1.01
+
```

Comment:
your call on this one, should these consts be kept in constants directory?

Reply:
Moved to constants file